### PR TITLE
UCT/IB: cm is back as default transport for connection establishment

### PR DESCRIPTION
**Pull request by alex-mikheev**
@yosefe @alinask 
reverting to cm till we solve ud issues

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1239/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/652/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/655/ for details (Mellanox internal link).

File: src/uct/ib/cm/cm_iface.c [hunk 1]
```diff
@@ -313,7 +313,7 @@ static ucs_status_t uct_cm_iface_query(uct_iface_h tl_iface,
     iface_attr->cap.am.max_bcopy      = mtu;
     iface_attr->iface_addr_len        = sizeof(uct_sockaddr_ib_t);
     iface_attr->ep_addr_len           = 0;
-    iface_attr->cap.flags             = /* UCT_IFACE_FLAG_AM_BCOPY | enable only in case of emergency */
+    iface_attr->cap.flags             = UCT_IFACE_FLAG_AM_BCOPY | /* enable only in case of emergency */
```
Comment (yosefe) [inline src/uct/ib/cm/cm_iface.c]: can we remove comment?
also - ud may be selected because it has lower latency. i think we need to change UD's flags

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1251/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/663/ for details (Mellanox internal link).

**Comment by shamisp** [comment]
autobot build

**Comment by shamisp** [comment]
:+1:

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/689/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1281/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/694/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/700/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1291/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1293/ for details.

**Comment by yosefe** [comment]
@alex-mikheev do you think UD is stable enough now so we can kill this PR ?

**Comment by yosefe** [comment]
in the meantime ud became stable enough to be used instead of cm.

**Pull request closed without merging**